R/FilterStateRange.R

```diff
@@ -295,13 +338,10 @@ RangeFilterState <- R6::R6Class( # nolint
       # Required for displaying ticks on the slider, can modify choices!
       if (identical(diff(x_range), 0)) {
         choices <- x_range
-        private$slider_ticks <- signif(x_range, digits = 10)
-        private$slider_step <- NULL
       } else {
-        x_pretty <- pretty(x_range, 100L)
+        x_pretty <- pretty(x_range, 10000L)
```

Comment:
Since we are no longer formatting/rounding, user can input as many sign digits as they like in the manual input, should not need tiny step size for the numeric input anymore. Let's revert to the original `100L` as the step size.

Reply:
Are you sure?

10000 steps:

https://github.com/insightsengineering/teal.slice/assets/114988527/6c4d13ed-d573-42ec-96e1-4de67b726905

100 steps:

https://github.com/insightsengineering/teal.slice/assets/114988527/6f9a77b1-5842-400d-aaf8-a681f7875e2e